internal/test/wmcb/wmcb_test.go

```diff
@@ -54,6 +56,16 @@ const (
 	hybridOverlayConfigurationTime = 2 * time.Minute
 	// hybridOverlayServiceName is the name of hybrid-overlay Windows service
 	hybridOverlayServiceName = "hybrid-overlay-node"
+	// containerdDir is the directory for storing the CNI plugins
+	containerdDir = payloadDirectory + containerdServiceName
```

Comment:
name doesnt match with description, please change to cniDir if description is accurate